LEAF_Request_Portal/js/formSearchMultisite.js

```diff
@@ -0,0 +1,1111 @@
+			    <img id="' +
+        prefixID +
+        'searchIconBusy" class="searchIcon" alt="loading" style="vertical-align: middle; padding-right: 4px; display:none" src="' +
+        rootURL +
+        'images/indicator.gif">\
+					<span style="position: absolute; width: 60%; height: 1px; margin: -1px; padding: 0; overflow: hidden; clip: rect(0,0,0,0); border: 0;" aria-atomic="true" aria-live="polite" class="status" role="status"></span>\
+			    <input style="border: 1px solid black; padding: 4px" type="text" id="' +
+        prefixID +
+        'searchtxt" name="searchtxt" size="50" title="Enter your search text" value="" />\
+			    <button class="buttonNorm" id="' +
+        prefixID +
+        'advancedSearchButton">Advanced Options</button>\
+			    <fieldset id="' +
+        prefixID +
+        'advancedOptions" style="position: relative; display: none; margin: 0px; border: 1px solid black; background-color: white">\
+		        <legend>Advanced Search Options</legend>\
+		        <button id="' +
+        prefixID +
+        'advancedOptionsClose" style="float: right; margin-top: -20px; margin-right: -14px; display: none; cursor: pointer; background-image:url(' +
+        rootURL +
+        'dynicons/?img=process-stop.svg&w=16); height: 16px;width: 16px; border: none; background-color: transparent; text-indent: -9999em" alt="Close advanced search">Close advanced search</button>\
+		        <div style="width: 550px">Find items where...</div>\
+		        <table id="' +
+        prefixID +
+        'searchTerms"></table>\
+		        <button class="buttonNorm" id="' +
+        prefixID +
+        'addTerm" style="float: left">And...</button>\
+		        <button class="buttonNorm" id="' +
+        prefixID +
+        'orTerm" style="float: left">Or...</button>\
+		        <br /><br />\
+		        <button id="' +
+        prefixID +
+        'advancedSearchApply" class="buttonNorm" style="text-align: center; width: 100%">Apply Filters</button>\
+		    </fieldset>\
+		    </div>\
+		    <div id="' +
+        prefixID +
+        '_result" style="margin-top: 8px" aria-label="Search Results">\
+		    </div>'
+    );
+
+    var searchOrigWidth = 0;
+    $("#" + prefixID + "advancedOptionsClose").on("click", function () {
+      localStorage.setItem(localStorageNamespace + ".search", "");
+      $("#" + prefixID + "searchtxt").val("");
+      search("");
+      $("#" + prefixID + "advancedOptionsClose").css("display", "none");
+      $("#" + prefixID + "advancedOptions").slideUp(function () {
+        $("#" + prefixID + "advancedSearchButton").fadeIn();
+        $("#" + prefixID + "searchtxt").css("display", "inline");
+        $("#" + prefixID + "searchtxt").animate(
+          { width: searchOrigWidth },
+          400,
+          "swing"
+        );
+        $("#" + prefixID + "searchtxt").focus();
+      });
+    });
+    //added for keyboard navigation and accessibility to close advanced search options
+    var searchOrigWidth = 0;
```

Comment:
This is the same as line 76 above.  And does this need to be a var?  Looks like it would be a case for let

Reply:
Nice catch. This was a copy/paste job. I'll update these here, but updating the other file should go into a different effort to reduce risk.